src/main/java/org/usfirst/frc/team2473/robot/subsystems/SparkDriveSubsystem.java

```diff
@@ -0,0 +1,250 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team2473.robot.subsystems;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+
+import org.usfirst.frc.team2473.framework.Devices;
+import org.usfirst.frc.team2473.robot.RobotMap;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import org.usfirst.frc.team2473.framework.CHS_SparkMax;
+
+/**
+ * This class contains all components of the robot necessary for driving.
+ */
+public class SparkDriveSubsystem extends Subsystem {
```

Comment:
We have two compatible drive subsystems, DriveSubsystem and SparkDriveSubsystem. Why aren't we simply replacing the existing DriveSubsystem with the Spark controllers?